### DISCUSSION TITLE
Design tokens for colors other than grey

### DISCUSSION THREAD

**Discussion by charleneyjh1** [twilio-labs/paste#1138] [Q&A] [answered]
Hi team! 

I wonder do we have design tokens for colors other than grey in code(like $color-red-60?) ? On the Paste site I only see design token for grey. 

<img width="552" alt="Screen Shot 2021-01-29 at 12 14 46 PM" src="https://user-images.githubusercontent.com/56697620/106317474-99e57680-622b-11eb-99eb-4917efebe332.png">

Thank you!

**Comment by SiTaggart** [comment, marked as answer]
Hi @charleneyjh1 

As per the introduction to Design Tokens on the doc site, our tokens are named based on their use case.

> For example $color-background-error is named to indicate the appropriate background color for an error, as opposed to $color-background-red which is less specific and could be used in a number of ways depending on cultural context and design language.
https://paste.twilio.design/tokens

We split colors into categories, [border](https://paste.twilio.design/tokens#border-colors), [backgrounds](https://paste.twilio.design/tokens#background-colors), [text colors](https://paste.twilio.design/tokens#text-colors), [shadows](https://paste.twilio.design/tokens#box-shadows), etc. and those are the only colors available for you to use. (I'm not even 100% sure you can use the greys anymore 😨 )

This is how we scale and translate design language to implementation.

New design tokens are added to system after rigorous consideration as it is unlikely we'll ever be able to remove them once they are added, so it's a big deal.

We're always keen to hear about new design challenges you face using the system, if you think there are missing use cases then please let us know.
